tests/testAllocationConfig.py

```diff
@@ -23,30 +23,40 @@
 import unittest
 import os.path
 from lsst.ctrl.execute.allocationConfig import AllocationConfig
+import lsst.utils.tests
 
```

Comment:
I assume these changes in `tests/` are not done by flake8; the commit message "flake8 code cleanup" is then confusing..